tests/dev/test_ci_uv_sync_diag.py

```
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+
+def _script_path() -> Path:
+    return Path(__file__).resolve().parents[2] / "scripts" / "dev" / "ci_uv_sync_diag.sh"
+
+
+def test_ci_uv_sync_diag_shell_syntax() -> None:
+    """Validate that the diagnostic helper passes bash syntax checks."""
+    script = _script_path()
+    assert script.exists(), "ci_uv_sync_diag.sh helper is missing"
+    assert subprocess.run(["bash", "-n", str(script)], check=False, timeout=30).returncode == 0
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)